irohad/consensus/gate_object.hpp

```diff
@@ -27,26 +27,39 @@ namespace iroha {
       consensus::Round round;
     };
 
-    /// Network votes for another pair and round
-    struct VoteOther {
+    struct Synchronizable {
       shared_model::interface::types::PublicKeyCollectionType public_keys;
-      shared_model::interface::types::HashType hash;
       consensus::Round round;
+      Synchronizable(
+          shared_model::interface::types::PublicKeyCollectionType _public_keys,
```

Comment:
Is this such a codestyle for struct constructor params? (names starting from  underscore)

Reply:
1. There is no any special style guideline for that case.
2. According to fast googling results, that is true - we can but that may cause false positive gcc warnings about unused variables.